index.bs

```diff
@@ -278,6 +278,35 @@ To <dfn abstract-op lt="set-user">set the `Sec-Fetch-User` header</dfn> for a [=
 </ol>
 </div>
 
+The `Sec-Purpose` HTTP Request Header {#sec-purpose-header}
+-----------------------------------------------------------------
+
+The <dfn http-header export>`Sec-Purpose`</dfn> HTTP request header exposes the purpose of the
+fetch, i.e. whether it's a [^link/rel/prefetch^|prefetch] or a regular fetch. It is a
+[=Structured Header=] whose value is a [=structured header/token=].
+[[!I-D.ietf-httpbis-header-structure]] Its ABNF is:
+
+```
+Sec-Purpose = sh-token
```

Comment:
The `sh-token` grammar does not support `prefetch; prerender`. I think you'd need to define this as an `sf-item` (https://www.rfc-editor.org/rfc/rfc8941.html#section-3.3) along with some processing instructions that note that `prerender` is the only valid parameter (https://www.rfc-editor.org/rfc/rfc8941.html#param).